app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] as part of the requirements the post model should have many comments and likes , kindly fix that 🙏